packages/sdk/src/config.ts

```diff
@@ -19,6 +19,7 @@ export class Config {
 
     if (environment === Environment.LOCAL) {
       this.environment = localConfig;
+      return;
```

Comment:
@nmlinaric @mpetrun5 @FSM1 There was a bug here, as it would always try to load shared config, even though you select `LOCAL` environment.